dissect/target/helpers/regutil.py

```diff
@@ -28,6 +29,19 @@
 """The possible value types that can be returned from the registry."""
 
 
+class RegValueType(Enum):
+    REG_NONE = regf.REG_NONE
+    REG_SZ = regf.REG_SZ
+    REG_EXPAND_SZ = regf.REG_EXPAND_SZ
+    REG_BINARY = regf.REG_BINARY
+    REG_DWORD = regf.REG_DWORD
+    REG_DWORD_BIG_ENDIAN = regf.REG_DWORD_BIG_ENDIAN
+    REG_MULTI_SZ = regf.REG_MULTI_SZ
+    REG_FULL_RESOURCE_DESCRIPTOR = regf.REG_FULL_RESOURCE_DESCRIPTOR
+    REG_RESOURCE_REQUIREMENTS_LIST = regf.REG_RESOURCE_REQUIREMENTS_LIST
+    REG_QWORD = regf.REG_QWORD
```

Comment:
```suggestion
    NONE = regf.REG_NONE
    SZ = regf.REG_SZ
    EXPAND_SZ = regf.REG_EXPAND_SZ
    BINARY = regf.REG_BINARY
    DWORD = regf.REG_DWORD
    DWORD_BIG_ENDIAN = regf.REG_DWORD_BIG_ENDIAN
    MULTI_SZ = regf.REG_MULTI_SZ
    FULL_RESOURCE_DESCRIPTOR = regf.REG_FULL_RESOURCE_DESCRIPTOR
    RESOURCE_REQUIREMENTS_LIST = regf.REG_RESOURCE_REQUIREMENTS_LIST
    QWORD = regf.REG_QWORD
```
Maybe? Putting `REG_` in front feels a bit double as that's how you usually do enums in C, but now we can properly use periods to access enum members.